plugins/inputs/system/ps.go

```diff
@@ -131,29 +137,22 @@ func (s *SystemPS) DiskUsage(
 		if hostMountPrefix != "" && !strings.HasPrefix(p.Mountpoint, hostMountPrefix) {
 			mountpoint = filepath.Join(hostMountPrefix, p.Mountpoint)
 			// Exclude conflicting paths
-			if paths[mountpoint] {
+			if paths.has(mountpoint) {
 				if s.Log != nil {
-					s.Log.Debug("[SystemPS] => dropped by mount prefix")
+					s.Log.Debugf("[SystemPS] => dropped by mount prefix (%q): %q", mountpoint, hostMountPrefix)
 				}
 				continue
 			}
 		}
-		if s.Log != nil {
-			s.Log.Debugf("[SystemPS] -> using mountpoint %q...", mountpoint)
-		}
 
 		du, err := s.PSDiskUsage(mountpoint)
 		if err != nil {
 			if s.Log != nil {
-				s.Log.Debugf("[SystemPS] => dropped by disk usage (%q): %v", mountpoint, err)
```

Comment:
I changed the message text to emphasize that there was an error

Reply:
Maybe we should use `s.Log.Errorf`?

Reply:
Probably so. It is a real error. Hiding it in a debug message is less useful to anyone who hits it.

Reply:
done in 4c3ed8d